pkgs/development/tools/misc/autoconf-archive/default.nix

```diff
@@ -13,6 +13,7 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   buildInputs = [ xz ];
+  nativeBuildInputs = lib.optionals stdenv.buildPlatform.isFreeBSD [ texinfo ];
```

Comment:
Why just on FreeBSD?

Reply:
See conversation with John on the main thread. The unqualified-uname problem which precipitates this is present for a few operating systems, but I believe FreeBSD is the only one we support building from. We _can_ enable this for all platforms and it will be harmless, but I think this signals the mechanism more clearly. I can also add a comment.

Reply:
what does `texinfo` have to do with `uname`?

Reply:
Patching the unqualified uname trips some modified-file check which causes some additional configure time logic which invokes texinfo. See above for the exact error message which explains this.

Reply:
done

Reply:
Ah, it finally clicked what the "modified-file check" means / how that matters. Got it.